src/network-manager/azext_network_manager/_params.py

```diff
@@ -536,3 +541,155 @@ def load_arguments(self, _):
         c.argument('flag', type=str, help='Default rule flag.')
         c.argument('kind', type=str, help='Required. Whether the rule is custom or default.Constant filled by server.', arg_type=get_enum_type(['Custom', 'Default']))
         c.ignore('user_rule')
+
+    with self.argument_context('network manager connection create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
```

Comment:
Instead of below a resource group, network manager connections are created at the subscription level. Below is an example resource group id: 
/subscriptions/{subscriptionId}/providers/Microsoft.Network/networkManagerConnections/{networkManagerConnectionName}